bundler/spec/plugins/install_spec.rb

```diff
@@ -32,7 +32,7 @@
   it "shows help when --help flag is given" do
     bundle "plugin install --help"
 
-    expect(out).to include("bundle plugin install PLUGINS    # Install the plugin from the source")
+    expect(out).to include("bundle plugin install PLUGINS")
```

Comment:
@tnir I think this failure is expected, because previously, since we had no man page, we defaulted to [Thor](https://github.com/rails/thor)'s autogenerated help text. We should be good updating the text to what the manual page prints now.

Reply:
Also if you're having trouble with the output, you might want to use the `with_fake_man` helper, which stubs the `man` executable to just print the argument given to it.

Reply:
This fails as `MANWIDTH` is `80` or so in the CI while that is 300 in my local. `bundle  plugin  install  PLUGINS` is included both Thor-based and the man introduced in this PR.

```
bundle  plugin  install  PLUGINS  [--source=SOURCE] [--version=version]
       [--git|--local_git=git-url] [--branch=branch] [--ref=rev]
```

As I have no idea on testing multiple spaces between words added by justification easily, I chose different parts from the updated help messages in the latest commit.

Reply:
That sounds fine. The other time I checked for this man output I checked the man header: https://github.com/rubygems/rubygems/blob/6b3a5a9ab0453463381a8164efb6298ea9eb776f/.github/workflows/install-rubygems.yml#L77